### ISSUE TITLE
Warrior Enrage Procs when getting hit while sitting

### ISSUE THREAD

**Issue by somnu** [LightsHope/issues#851]
The issue: Warrior Enrage talent can be "forced" into proccing by /sit and getting hit.

How it should work: Enrage should never proc when getting hit while /sitting.

The Enrage talent didn't work like that on retail vanilla. Although getting hit while /sit would yield a critical hit on your character, it did not yield procs like enrage.
Proof:

http://wowwiki.wikia.com/wiki/Sit?diff=360741&oldid=294765    from:  03:51, December 10, 2006 (view page history)‎

"Melee attacks against a sitting target automatically hit and cause double damage. The first damage taken while sitting will cause a target to immediately stand. The double damage registers as a critical hit in the combat log, but does not trigger abilities as a normal crit would (such as the Fury Warrior talent: Enrage)"﻿

https://wow.gamepedia.com/Patch_2.3.0
It seems they added this mechanic in 2.3.0 (TBC).

"[Enrage]: This talent can now trigger while the Warrior is sitting."

This implies that before 2.3.0 it didn't work.

How to reproduce: Make a warrior, spec into enrage and do /sit in front of a mob or player and make them attack you.

**Comment by banallwowbots** [comment]
Same goes for Blood Craze, Reckoning talent for paladins, Blessed Recovery talent for priests and other "struck by critical hit"-talents

These talents should not be triggered from auto attacks or npc attacks **while sitting**. The only exception is if a player crits with an ability (while sitting), like Heroic Strike or Aimed Shot.

**Comment by cudderi** [comment]
Here is a vanilla wow video of a paladin stacking Reckoning by sitting repeatedly. https://www.youtube.com/watch?v=kuPEzEoexLI
The video is filmed after patch 1.9, since he has 15min buffs on bars, and the video is uploded 8.8.2006.

Another video with the same upload date where he is reck-bombing players, but the patch is 1.11, since EPL has guards. However, the paladin has slightly different gear, so they might not be filmed close to each other. https://www.youtube.com/watch?v=LDJX0oT3PlU

Edit: Seems like the first link is invalid proof for stacking reckoning stacks,(he gets crit but doesn't get a reckoning stack) and thus the second link is also useless. Altough he gets 4 stacks of reckoning with 13 attacks of which 7 were critical, its not that unlikely that all of the reckoning ones were normal crits.

**Comment by Scyllaqt** [comment]
Just to link another source - http://blue.cardplace.com/cache/wow-paladin/1069149.htm

"Charges will not generate off auto attacks or npc attacks by trying to sit down and force a crit. However, ability crits from physical abilities such as Sinister Strike, Hamstring, Auto-shot, Aimed shot, etc will generate a charge if you're sitting." 

However, the video posted by cudi directly contradicts this source.  Considering all the conflicting evidence it's likely this is something that was changed at some point in vanilla, or sit crits could proc some buffs but not others.

Edit: Also, before anyone claims that the wowwiki revision in the OP should be disregarded due to its date (post 2.0), the [October 2006 Revision](http://wowwiki.wikia.com/wiki/Sit?oldid=294765) says the same thing.

**Comment by somnu** [comment]
Yes the only proc I am sure of is Enrage because it is something I have actually tested back in 2005 when I was playing warrior and trying to find out new ways to abuse my class.

**Comment by banallwowbots** [comment]
Sorry somnu for discussing this in your issue post. I should probably make another post, but I feel this is related anyhow. 

cudderi:
Regarding your video (https://www.youtube.com/watch?v=kuPEzEoexLI), please do a count on critical hits received and compare it to Reckoning charges. What did you find?

Unfortunately I could not find anything related to this issue, in your 2nd video.

Edit: Created a new issue regarding Reckoning Charges: [here](https://github.com/LightsHope/issues/issues/854)

**Comment by theloras** [comment]
Enrage and other "on offensive crits against you" abilities became broken multiple times in TBC and were fixed as a result each time:

"Fixed an issue where spells that proc while a player was standing would not proc if they were sitting or lying down."
http://wowwiki.wikia.com/wiki/Patch_2.0.1

"Enrage: This talent can now trigger while the Warrior is sitting."
http://wowwiki.wikia.com/wiki/Patch_2.3.0

**Comment by bugabuser** [comment]
https://www.youtube.com/watch?v=drbJ1TaQH64&feature=youtu.be&t=28 

Note the warrior gets hit by the bear and doesn't get enrage. Later in the clip, he gets crit and gets enraged, so it's confirmed he has the talent

**Comment by sjnfnd** [comment]
Where in later in the clip does he get enrage? He didnt get enrage because he doesnt have the talent

**Comment by hasselz** [comment]
Do spell criticals trigger enrage from warrior? I don't think spell crits should trigger it but i cannot remember it. I have seen it proc from spells on some servers

**Comment by theloras** [comment]
This bug report is a moot point for the simple fact that Blizzard "FIXED" these talents to be able to proc while sitting in 2.0.1:

**"Fixed an issue where spells that proc while a player was standing would not proc if they were sitting or lying down."**
_http://wowwiki.wikia.com/wiki/Patch_2.0.1_

**"Enrage: This talent can now trigger while the Warrior is sitting."**
_http://wowwiki.wikia.com/wiki/Patch_2.3.0_

Now, why is this of critical importance on Lightshope...cause the LH Devs specifically made a decision to put in a non-Vanilla change that was considered a "fix" in 2.0.1:

**Warrior "Shield Slam" and "Shield Bash" will no longer trigger a proc from an equipped weapon.**
_http://wowwiki.wikia.com/wiki/Patch_2.0.1_

If The Shield Slam/Bash issue listed in the 2.0.1 Patch Notes under "Bugs" is considered to be relevant for Lightshope and its version of Vanilla WoW, then they all have to be considered de facto.

TLDR - you can close this bug report now.

/thread

**Comment by sjnfnd** [comment]
Theloras what even is your existence

**Comment by somnu** [comment]
> This bug report is a moot point for the simple fact that Blizzard "FIXED" these talents to be able to proc while sitting in 2.0.1:
> 
> **"Fixed an issue where spells that proc while a player was standing would not proc if they were sitting or lying down."**
> _http://wowwiki.wikia.com/wiki/Patch_2.0.1_
> 
> **"Enrage: This talent can now trigger while the Warrior is sitting."**
> _http://wowwiki.wikia.com/wiki/Patch_2.3.0_
> 
> Now, why is this of critical importance on Lightshope...cause the LH Devs specifically made a decision to put in a non-Vanilla change that was considered a "fix" in 2.0.1:
> 
> **Warrior "Shield Slam" and "Shield Bash" will no longer trigger a proc from an equipped weapon.**
> _http://wowwiki.wikia.com/wiki/Patch_2.0.1_
> 
> If The Shield Slam/Bash issue listed in the 2.0.1 Patch Notes under "Bugs" is considered to be relevant for Lightshope and its version of Vanilla WoW, then they all have to be considered de facto.
> 
> TLDR - you can close this bug report now.
> 
> /thread

I hate to link to a tipsout video, but get rekt scum: https://youtu.be/fTnFuUUXHYk?t=716
